### PR TITLE
Added ImageMagick and Assimp finders

### PR DESCRIPTION
**Pull request by favreau**

### CODE REVIEW

File: FindImageMagick.cmake [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(
+    imagemagick DEFAULT_MSG
+    IMAGEMAGICK_LIBRARIES
+    IMAGEMAGICK_INCLUDE_DIRS)
+
+if(IMAGEMAGICK_FOUND)
+  set(IMAGEMAGICK_INCLUDE_DIRS ${IMAGEMAGICK_INCLUDE_DIR})
+  message(STATUS "Found ImageMagick in ${IMAGEMAGICK_INCLUDE_DIR}:${IMAGEMAGICK_LIBRARIES}")
```
Comment (tribal-tec) [inline FindImageMagick.cmake]: Please respect QUIET accordingly. Also, DEFAULT_MSG to find_package_handle_standard_args should make this print obsolete, no?

File: Findassimp.cmake [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+)
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(ASSIMP DEFAULT_MSG
+    ASSIMP_INCLUDE_DIRS
+    ASSIMP_LIBRARIES)
+
+if(ASSIMP_FOUND)
+  set(ASSIMP_INCLUDE_DIRS ${ASSIMP_INCLUDE_DIR})
+  message(STATUS "Found assimp in ${ASSIMP_INCLUDE_DIR}:${ASSIMP_LIBRARIES}")
```
Comment (tribal-tec) [inline Findassimp.cmake]: dito

**Comment by eile** [comment]
-1: ImageMagick has a .pc packageconfig file. common_package should suffice.

**Comment by eile** [comment]
Same for assimp: ```/opt/local/lib/pkgconfig/assimp.pc```

**Comment by favreau** [comment]
Thanks for the guidance, I'll close the PR.

**Pull request closed without merging**